ui/components/app/assets/asset-list/asset-list.js

```diff
@@ -213,6 +228,20 @@ const AssetList = ({ onClickAsset, showTokensLinks }) => {
       {showDetectedTokens && (
         <DetectedToken setShowDetectedTokens={setShowDetectedTokens} />
       )}
+      {showReceiveModal && (
+        <ReceiveModal
+          address={selectedAccount?.address}
```

Comment:
`ReceiveModal` requires a `selectedAccount?.address`?  I think we should add that as part of the condition for displaying in (on line 231)

Reply:
Nice, updated